resources/TrialApp.didact.md

```diff
@@ -1,16 +1,16 @@
 # Welcome to your Trial App Didact Tutorial
 
-This guide will show you how to make and deploy changes to your Azure App Service Trial App. Your trial will last 60 minutes. You can find more information about Azure App Service [here](https://docs.microsoft.com/en-us/azure/app-service/).
+This tutorial will show you how to make and deploy changes to your Azure App Service trial app. Your trial will last 60 minutes. You can find more information about Azure App Service [here](https://docs.microsoft.com/en-us/azure/app-service/).
```

Comment:
nit: I think whenever we use Azure App Service in the name, we should probably proper case it since it's like the "official" product name.